app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss

```diff
@@ -46,7 +46,7 @@
 
     &:hover,
     &:focus {
-      color: darken(govuk-organisation-colour("office-of-the-leader-of-the-house-of-commons"), 10%);
+      color: govuk-colour("black");
```

Comment:
I suggest we use `$govuk-focus-text-colour`, just in case we'll change the colours again 😁